src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/ExceptionTelemetryConverter.cs

```diff
@@ -17,7 +33,21 @@ public class ExceptionTelemetryConverter : CustomTelemetryConverter<ExceptionTel
         /// <returns>Telemetry entry to emit to Azure Application Insights</returns>
         protected override ExceptionTelemetry CreateTelemetryEntry(LogEvent logEvent, IFormatProvider formatProvider)
         {
+            Guard.NotNull(logEvent, nameof(logEvent), "Requires a Serilog log event to create an exception telemetry entry");
+            Guard.For(() => logEvent.Exception is null, new ArgumentException(
+                "Requires a log event that tracks an exception to create an exception telemetry entry", nameof(logEvent)));
+            
             var exceptionTelemetry = new ExceptionTelemetry(logEvent.Exception);
+
+            Type exceptionType = logEvent.Exception.GetType();
+            PropertyInfo[] exceptionProperties = exceptionType.GetProperties(BindingFlags.Public | BindingFlags.Instance);
```

Comment:
I'm not a reflection expert but can we omit the inherited properties from the exception? If so, I'd make this configurable because me personally would not like to have them as the Exception telemetry already provides these (think message, stack trace, etc)

Reply:
Ah, yest that would be better. Yeah, just adding `BindingFlags.DeclardOnly` will do that.

Reply:
I'd love to have that, thanks